examples/imagenet/train_benchmark.py

```diff
@@ -0,0 +1,62 @@
+"""Benchmark for the ImageNet example."""
+import tempfile
+
+import time
+from absl import flags
+from absl.testing import absltest
+from absl.testing.flagsaver import flagsaver
+
+import jax
+import numpy as np
+from flax.testing import Benchmark
+
+import train
+
+
+# Parse absl flags test_srcdir and test_tmpdir.
+jax.config.parse_flags_with_absl()
+
+
+FLAGS = flags.FLAGS
+
+
+class ImagenetBenchmark(Benchmark):
+  """Benchmarks for the ImageNet Flax example."""
+
+  @flagsaver
+  def test_8x_v100_half_precision(self):
+    """Run ImageNet on 8x V100 GPUs in half precision for 2 epochs."""
+    model_dir = tempfile.mkdtemp()
+    FLAGS.batch_size = 2048
+    FLAGS.half_precision = True
+    FLAGS.loss_scaling = 256.
+    FLAGS.num_epochs = 2
```

Comment:
With this way of benchmarking, I assumed that we go for full training, don't we?

Reply:
Yes, eventually. But not while setting up the system / debugging.